code/game/objects/items/stacks/stack_recipes.dm

```diff
@@ -403,7 +403,7 @@ var/list/datum/stack_recipe/wood_recipes = list (
 	new/datum/stack_recipe("item handle",		/obj/item/item_handle,					1,2,20,	time = 2 SECONDS							),
 	new/datum/stack_recipe("sword handle",		/obj/item/sword_handle,					1,2,10,	time = 2 SECONDS,							other_reqs = list(/obj/item/stack/sheet/metal = 1)),
 	new/datum/stack_recipe("wooden paddle",		/obj/item/weapon/macuahuitl,			1,		time = 50									),
-	new/datum/stack_recipe("baseball bat",		/obj/item/weapon/baseball_bat,			1,		time = 5 SECONDS							),
+	new/datum/stack_recipe("baseball bat",		/obj/item/weapon/baseball_bat,			15,		time = 8 SECONDS							),
```

Comment:
15 planks for a single bat is too much, 7-10 makes it less spammable but still common enough